services/autoscaling/tests/manual/.env-devel

```diff
@@ -0,0 +1,22 @@
+AUTOSCALING_DEBUG=true
```

Comment:
For this I have a proposal. Instead of having `.env-devel`-lke files (sort of a template .env) all around, every service could have a CLI command to generate this file. 

It is common practice e.g. in pylint or cookie-cutter to have a command to produce some kind of starting-point configuration. I created some tooling to integrate this uniformly and fast in the CLI of our services. Check e.g. https://github.com/ITISFoundation/osparc-simcore/blob/f2c5bb0bc240f6e11ca6732bf4bd4f841032f74a/services/payments/src/simcore_service_payments/cli.py#L29-L38


```cmd
$ simcore-service-autoscaling echo-dotenv > .env
$ cat .env
$ set -o allexport; source .env; set +o allexport
# To test the envs just 
$ simcore-service-autoscaling settings 
```

At some point we can even move part of the common code to `settings_library.utils_cli` as some other commands.

Reply:
well on one hand I agree, but on the other that means yet another step to go through in order to do manual testing.

Reply:
You can incorporate as a pre-condition in the Makefile dependencies. 
In addition, it will help keeping the manual tests up-to-date